### ISSUE TITLE
AEM 6.3: ResourceServiceManager and use of admin resource resolvers

### ISSUE THREAD

**Issue by joerghoh** [Adobe-Consulting-Services/acs-aem-commons#1059]
When deploying the current master branch (3.9.1-SNAPSHOT) on AEM 6.3 I get this exception:

```28.07.2017 20:10:28.180 *ERROR* [FelixStartLevel] com.adobe.acs.commons.replication.packages.automatic.impl.ConfigurationUpdateListener Exception allocating resource resolver
org.apache.sling.api.resource.LoginException: Bundle com.adobe.acs.acs-aem-commons-bundle is NOT whitelisted
...
        at org.apache.sling.resourceresolver.impl.CommonResourceResolverFactoryImpl.getResourceResolverInternal(CommonResourceResolverFactoryImpl.java:262)
        at org.apache.sling.resourceresolver.impl.CommonResourceResolverFactoryImpl.getAdministrativeResourceResolver(CommonResourceResolverFactoryImpl.java:151)
        at org.apache.sling.resourceresolver.impl.ResourceResolverFactoryImpl.getAdministrativeResourceResolver(ResourceResolverFactoryImpl.java:118)
        at com.adobe.acs.commons.replication.packages.automatic.impl.ConfigurationUpdateListener.getResourceResolver(ConfigurationUpdateListener.java:88)
        at com.adobe.acs.commons.replication.packages.automatic.impl.ConfigurationUpdateListener.getResourceResolver(ConfigurationUpdateListener.java:146)
        at com.adobe.acs.commons.util.ResourceServiceManager.refreshCache(ResourceServiceManager.java:139)
```
At this case we still use repository.getAdministrativeResourceResolver(), which requires an explicit whitelisting. If we don't want to do this, we should replace this call with a service user.

**Comment by davidjgonzalez** [comment]
Serendipitously @dklco fixed this in #1057 .. I think Travis failed tho :(

**Comment by davidjgonzalez** [comment]
Fixed via a71a645bcacaeb5b482506349166df5a26c95e19
